Source/WebCore/html/HTMLTableCellElement.cpp

```diff
@@ -59,7 +59,10 @@ unsigned HTMLTableCellElement::colSpan() const
 
 unsigned HTMLTableCellElement::rowSpan() const
 {
-    // FIXME: a rowSpan equal to 0 should be allowed, and mean that the cell is to span all the remaining rows in the row group.
+    unsigned rowSpanValue = rowSpanForBindings();
+    // when rowspan=0, the specs says it should apply to the full rows.
+    if (rowSpanValue == 0)
```

Comment:
How do I fix this?

> 'Tests for true/false, null/non-null, and zero/non-zero should all be done without equality comparisons.'

The stylechecker is not happy. But I found many places in the code with the same syntax.

Reply:
`if (!rowSpanValue)` works?

Reply:
```
if (!rowSpanValue)
    return maxRowSpan;
```

Should work. And maybe we should reuse `rowSpanValue` instead of calling `rowSpanForBindings()` again below in `return std::max(1u, rowSpanForBindings());`?

Reply:
Done! Thanks a lot @Ahmad-S792 and @twilco